katsdptelstate/test/test_rdb_handling.py

```diff
@@ -111,7 +111,7 @@ def load_from_file_and_check(self, file):
         # Load RDB file back into some backend
         read_ts = self.make_telescope_state()
         read_ts.load_from_file(file)
-        self.assertEqual(read_ts.keys(), [b'immutable', b'mutable'])
+        self.assertEqual(read_ts.keys(), ['immutable', 'mutable'])
```

Comment:
I would cherry-pick this commit onto master so long to fix it.

Reply:
Done.